paper/paper.md

```
@@ -0,0 +1,102 @@
+title: 'A Snakemake workflow for differential expression analyis with alternative splicing detection using long read data'
+tags:
+  - Snakemake
+  - Nanopore
+  - HPC
+  - differential gene expression
+  - alterternative splicing detection
+authors:
+  - name: Yannic Eising
+    orcid: 0009-0003-9103-5689
+    affiliation: "1, 2" # (Multiple affiliations must be quoted)
+  - name: Sören Lukas Hellmann
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)